automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala

```diff
@@ -40,7 +40,8 @@ case class ClusterRequest(labels: LabelMap = Map(),
                           rstudioDockerImage: Option[String] = None,
                           welderDockerImage: Option[String] = None,
                           scopes: Set[String] = Set.empty,
-                          enableWelder: Option[Boolean] = None)
+                          enableWelder: Option[Boolean] = None,
+                          customClusterEnvironmentVariables: Option[Map[String, String]] = None)
```

Comment:
1. this is probably not the place you want to make the change. https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala#L87 might be the place you need

2. Use `Map[String, String]` instead of wrapping in `Option` since an empty `Map` already has represents the idea of no custom environment variable